triton-client/src/store/thunks.ts

```diff
@@ -52,64 +52,49 @@ export const fetchProjects =
 export const fetchRuns =
     (externalProjectId: ExternalProjectID) =>
     async (dispatch: AppDispatch, getState: () => RootState) => {
-        try {
-            const runs = await apiTriton.listRunsForProjects([
-                externalProjectId,
-            ])
-            // console.info('Loaded runs succesfully', runs)
-            dispatch(RunsStateActions.setRuns(runs))
-            // console.info(getState().runsState.runsById)
-            return runs
-        } catch (err: any) {
-            throw err
-        }
+        const runs = await apiTriton.listRunsForProjects([externalProjectId])
+        // console.info('Loaded runs succesfully', runs)
+        dispatch(RunsStateActions.setRuns(runs))
+        // console.info(getState().runsState.runsById)
+        return runs
     }
 
 export const fetchDatasets =
     (runName: TritonRun["name"]) =>
     async (dispatch: AppDispatch, getState: () => RootState) => {
-        try {
-            const run = getState().runsState.runsByName[runName]
-            if (!run) {
-                return []
-            }
+        const run = getState().runsState.runsByName[runName]
+        if (!run) {
+            return []
+        }
 
-            // console.info(datasetIds)
-            const datasets = await apiTriton.listDatasetsByIds(run.datasets)
-            // console.debug(`Loaded datasets succesfully: ${JSON.stringify(datasets)}`)
-            dispatch(DatasetsStateActions.setDatasets(datasets))
+        // console.info(datasetIds)
+        const datasets = await apiTriton.listDatasetsByIds(run.datasets)
+        // console.debug(`Loaded datasets succesfully: ${JSON.stringify(datasets)}`)
+        dispatch(DatasetsStateActions.setDatasets(datasets))
 
-            return datasets
-        } catch (err: any) {
-            throw err
-        }
+        return datasets
     }
 
 export const fetchRequests =
     (datasetIds: Array<TritonDataset["id"]>) =>
     async (dispatch: AppDispatch, getState: () => RootState) => {
-        try {
-            const requests =
-                await apiTriton.listRequestsByDatasetIds(datasetIds)
-            // console.debug(`Loaded datasets succesfully: ${JSON.stringify(datasets)}`)
-            dispatch(RequestsStateActions.setRequests(requests))
-            const projectIDs: Set<string> = new Set()
-            for (const datasetId of datasetIds) {
-                const projectId =
-                    getState().datasetsState.datasetsById[datasetId]
-                        ?.external_project_id
-                if (projectId) {
-                    projectIDs.add(projectId)
-                }
+        const requests = await apiTriton.listRequestsByDatasetIds(datasetIds)
```

Comment:
i got confused with the name of this variable, definitely would be more intuitive to change the name of this type of db model

Reply:
i also dont think that removing try catches is the right solution to solving the error handling situation. maybe expanding the error handler will be better because would still be able to see what went wrong with the try/catch